reproducer/add_artifact_diffs.py

```diff
@@ -0,0 +1,106 @@
+import argparse
```

Comment:
Adding artifact diffs is out of scope for ActionsRemaker. This file should be removed.